src/audio/lib/data.c

```diff
@@ -686,6 +686,7 @@ f32 gDefaultPanVolume[] = {
     0.086471f, 0.074143f, 0.061803f, 0.049454f, 0.037097f, 0.024734f, 0.012368f, 0.0f,
 };
 
+ALIGNED(16)
 s16 gLowPassFilterData[16 * 8] = {
     /* 0x0 */ 0,     0,     0,    32767, 0,    0,     0,     0, // Identity filter (delta function)
```

Comment:
iirc we discovered a formatter option that allows macros with parens to stay on the same line as other stuff (@Dragorn421 ?)

Reply:
yeah we did it for BAD_RETURN `TypenameMacros: ['BAD_RETURN']` in .clang-format

Reply:
Adding it to `TypenameMacros` didn't change the formatting unfortunately. I've fixed these manually for the time being

Reply:
its just going to be a problem the next time someone wants to run the formatter, right.
wonder why it doesnt work

Reply:
Oh I thought I'd done it by mistake and the formatter was just not doing anything lol

Turns out adding it to TypenameMacros prevents the formatter from reverting it, but didn't prompt the formatter to move it into the right place.